### PR TITLE
To Do List in React

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
Transformed the To-Do List based on HTML, CSS, and JavaScript to To-Do List in React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,274 @@
...
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+- Mobile First Approach
+- Responsive Website
+- Button Interactions (i.e. hover, etc.)
+- Attractive Images & Design
+- Modal/dialog
+- Dynamic page (data is retrieved from JSON file)
```
Comment (mnaibei) [inline README.md:145]: Hi @RHK-MICROVERSE 
Great job working on this project. It looks good.
I have a few suggestions on how to make it better: 👇🏾
 

Please adjust your key features list on README to make it current

Comment (mnaibei) [inline README.md:145]: Hi @RHK-MICROVERSE 
Great job working on this project. It looks good.
I have a few suggestions on how to make it better: 👇🏾

Comment (mnaibei) [reply README.md:145]: ### SUGGESTIONS
Please adjust the list of your key features to reflect the scope of your current project

File: README.md [hunk 2]
```diff
@@ -1 +1,274 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo Link](https://github.com/RHK-MICROVERSE/todolistreact/issues)
```
Comment (mnaibei) [inline README.md:151]: Please remove this live demo link because you have not deployed the project

**Review by mnaibei — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mnaibei) [inline README.md]: Please adjust your key features list to better reflect the scope of the current project

File: README.md (hunk 2 above)
Comment (mnaibei) [inline README.md]: Please remove the live demo link as you have not deployed the project or deploy it first then add the link

**Review by mnaibei — CHANGES_REQUESTED** [review]
Hi @RHK-MICROVERSE
Great job working on this project. It looks good.

### **Highlights**
✔️ Gitflow was correctly used
✔️ Good design
✔️  App works as expected

I have a few suggestions on how to make it better: 👇🏾


# **Required Changes ♻️**
_Check the comments under the review._

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: README.md (hunk 1 above)
Comment (RHK-MICROVERSE) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (RHK-MICROVERSE) [reply README.md]: Done

**Review by mnaibei — APPROVED** [review]
Hello @RHK-MICROVERSE 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

Congratulations! 🎉
Highlights
Good project structure ✔️
Professional readme file ✔️
No linter errors ✔️
Amazing design ✔️



# Status: Approved ✔️✔️✔️
Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @mnaibei  in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**